dev_deps.bzl

```diff
@@ -0,0 +1,44 @@
+"dev dependencies needed to develop rules_proto"
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+
+def rules_proto_dev_deps():
+    http_archive(
+        name = "rules_cc",   
+        sha256 = "4aeb102efbcfad509857d7cb9c5456731e8ce566bfbf2960286a2ec236796cc3",
+        strip_prefix = "rules_cc-2f8c04c04462ab83c545ab14c0da68c3b4c96191",
```

Comment:
Please depend on a released version of rules_cc (possibly preexisting).

Url links like `https://github.com/bazelbuild/rules_cc/archive/2f8c04c04462ab83c545ab14c0da68c3b4c96191.tar.gz` are not guaranteed by github to have stable hashes. If github decides to change how they archive commits, this breaks the dependency.

Reply:
that would be out of scope for this PR as this is an existing dependency I just moved around.

Reply:
> It's still left in WORKSPACE file, please remove it then there.

Sorry, I don't follow, what is left on the WORKSPACE, the `rbe_preconfig` call? I don't think I can do that as bazel doesn't allow loading from a workspace in the same macro call.

Reply:
I’m advocating for removing dependency on protobuf repo and removing versions from rules_proto.

I’m not sure on which of those two points you disagree